policybot/handlers/githubwebhook/filters/labeler/labeler.go

```diff
@@ -251,9 +275,9 @@ func (l *Labeler) matchAutoLabel(al config.AutoLabel, title string, body string,
 		return false
 	}
 
-	// if any labels match, we're done
+	// if the labels don't match, we're done
 	for _, label := range labels {
-		if l.labelMatch(al, label.LabelName) {
+		if !l.labelMatch(al, label.LabelName) {
```

Comment:
Since I'm not familiar with this code, not sure what this is doing. But from a glance, it seems to not do what the comment is saying. Doesn't `al` need to fail match against all of the labels, rather than just one?

Reply:
Oh, ouch. Good  catch.

I rewrote that now, should be much clearer, and correct.